baybe/constraints/conditions.py

```diff
@@ -33,12 +36,31 @@ def _is_not_close(x: ArrayLike, y: ArrayLike, rtol: float, atol: float) -> np.nd
     return np.logical_not(np.isclose(x, y, rtol=rtol, atol=atol))
 
 
+def isclose(x: PolarsType, y: PolarsType, rtol: float, atol: float) -> Iterable[bool]:
+    """Return a boolean array indicating where ``x`` and ``y`` are close.
+
+    The equivalent to ``numpy.isclose``.
+
+    Args:
+        x: First input array to compare.
+        y: Second input array to compare.
+        rtol: The relative tolerance parameter.
+        atol: The absolute tolerance parameter.
+
+    Returns:
+        Returns a boolean array of where ``x`` and ``y`` are equal within the
+        given tolerances.
+
+    """
+    return ops.le(abs(x - y), atol + rtol * abs(y))
```

Comment:
line 55 still has a mypy error:` baybe/constraints/conditions.py:55: error: Unsupported operand types for * ("float" and "object")  [operator]`
Is it ok to ignore this line? I appreciate any suggestions to fix this.

Reply:
I think this might stem from some imprecise typing of your inputs. In fact, not sure if something like `PolarsType` is even required here. And if it is, does numpy's `ArrayLike` work together with polars? Would doubt it, actually... 🤔 So what is really that this function now gets as inputs during runtime? Are we passing polars, numpy, both?

Reply:
ArrayLike had worse issues with mypy:
```
 Unsupported operand types for - ("_SupportsArray[dtype[Any]]" and "bool")  [operator]
 Unsupported operand types for - ("_SupportsArray[dtype[Any]]" and "int")  [operator]
 Unsupported operand types for - ("_SupportsArray[dtype[Any]]" and "float")  [operator]
 Unsupported operand types for - ("_SupportsArray[dtype[Any]]" and "complex")  [operator]
 .....
```
I believe the problem is similar to what we had in `np.close`:

```TypeError: ufunc 'isfinite' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe'' ```
I think Polars doesn't pass variables solely as numerical types, since we can have any type of values in the columns.

Reply:
Ok I found the culprit: I should have used numpy operators instead of ordinary ones, e..g. `np.subtract` and `np.abs`. Fixed.